addons/website/static/src/services/website_service.js

```diff
@@ -98,6 +106,9 @@ export const websiteService = {
             async sendRequest(route, params, readMethod = "text", method = "post") {
                 return http[method](route, { ...params, 'csrf_token': core.csrf_token }, readMethod);
             },
+            toggleFullscreen() {
+                document.body.classList.add('o_website_fullscreen');
+            }
```

Comment:
This always applies on the global external document ? I believe it does - just to make sure.
Missing `,` after `}`.